src/js/weather.js

```diff
@@ -0,0 +1,34 @@
+import { DailyWeather } from './daily-weather';
+
+export class Weather {
+    constructor(location) {
+        this.location = location;
+        this.forecast = [];
+    }
+
+    async loadWeather() {
+        let response = await fetch(`https://api.openweathermap.org/data/2.5/onecall?lat=${this.location.latitude}&lon=${this.location.longitude}&lang=${this.location.language}&exclude=minutely,hourly,alerts&appid=afba4e3563e6d79fdbe184a899275267`);    
+        response = await response.json();
```

Comment:
at least
const { current, daily } = response.json();
always recheck do you really need whole bunch of data

Reply:
fixed